changelogs/fragments/1066-Mvs_to_non_existent_mvs_copy_destination_attrs_match_up.yml

```
@@ -0,0 +1,5 @@
+bugfixes:
+  - zos_copy - Module calculated incorrectly the attributes of the source dataset
```

Looks good to me, I would just change this changelog fragment a but to better describe the problem, use "data set" instead of "dataset" and also changed the word "target" to "destination" bc that is the word we used in our docs(is based on the option name dest)

```suggestion
  - zos_copy - When copying an executable data set with aliases and destination did not exist, destination data set was created with wrong attributes. Fix now creates destination data set with the same attributes as the source.
```

---

updated thanks